index.js

```diff
@@ -77,16 +101,35 @@ var parseRSS1 = function(xmlObj, callback) {
 var parseRSS2 = function(xmlObj, callback) {
   var json = {feed: {entries: []}};
   var channel = xmlObj.rss.channel[0];
-  if (channel['atom:link']) json.feed.feedUrl = channel['atom:link'][0].href;
+  if (channel['atom:link']) json.feed.feedUrl = channel['atom:link'][0].$.href;
   TOP_FIELDS.forEach(function(f) {
     if (channel[f]) json.feed[f] = channel[f][0];
   })
+  if (channel['itunes:owner']) {
```

Comment:
If possible, maybe we can do a top-level check to see if it's an itunes feed and extract all the itunes logic into a separate function